src/LfMerge.Core.Tests/SRTestEnvironment.cs

```
@@ -51,6 +55,58 @@ public SRTestEnvironment(TemporaryFolder? tempFolder = null)
 			Settings.CommitWhenDone = true; // For SR tests specifically, we *do* want changes to .fwdata files to be persisted
 		}
 
+		override protected void RegisterMongoConnection(ContainerBuilder builder)
+		{
+			// E2E tests want a real Mogno connection
+			builder.RegisterType<MongoConnection>().As<IMongoConnection>().SingleInstance();
+		}
+
+		public void LaunchMongo()
```

it looks like this method will mostly fail silently making tests fail in unexpected ways. This method should throw if it can't launch mongo

---

Done in commit 48f4608.